native_client/javascript/Makefile

```diff
@@ -3,47 +3,60 @@ NODE_ABI_TARGET    ?=
 NODE_BUILD_VERBOSE ?= --verbose
 PROJECT_NAME       ?= deepspeech
 PROJECT_VERSION    ?= $(shell cat ../../VERSION | tr -d '\n')
+NPM_ROOT           ?= $(shell npm root)
 
 include ../definitions.mk
 
 ifeq ($(TARGET),host-win)
 LIBS := '$(shell cygpath -w $(subst .lib,,$(LIBS)))'
 endif
 
+.PHONY: npm-dev
+
 default: build
 
 clean:
-	rm -f deepspeech_wrap.cxx package.json
+	rm -f deepspeech_wrap.cxx package.json package-lock.json
 	rm -rf ./build/
 
 clean-npm-pack:
-	rm -fr ./node_modules/
 	rm -fr ./deepspeech-*.tgz
 
 really-clean: clean clean-npm-pack
+	rm -fr ./node_modules/
 	rm -fr ./lib/
 
 package.json: package.json.in
 	sed \
 		-e 's/$$(PROJECT_NAME)/$(PROJECT_NAME)/' \
 		-e 's/$$(PROJECT_VERSION)/$(PROJECT_VERSION)/' \
-		package.json.in > package.json && cat package.json
+		package.json.in > package.json && cat package.json && pwd
 
-configure: deepspeech_wrap.cxx package.json
+npm-dev: package.json
+	# node-gyp@5.x behaves erratically with VS2015 and MSBuild.exe detection
+ifeq ($(TARGET),host-win)
+	npm install node-gyp@4.x
+endif
+	npm --no-color install --prefix=$(NPM_ROOT)/../ --ignore-scripts --force --verbose --production=false .
```

Comment:
This ensures we install `dependencies` and `devDependencies`

Reply:
Removed that, it was required when I was trying to fix something to avoid messing of control chars.